### PR TITLE
Fixed GRPC problem

### PR DESCRIPTION
**Pull request by caroline-church**
Rebuild the GRPC module on first activation

contributes to https://github.ibm.com/IBM-Blockchain/fabric-vscode-extension/issues/36

Signed-off-by: Caroline Church <caroline.church@uk.ibm.com>

### CODE REVIEW

File: .travis/rewritePackageJson.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+'use strict';
+
+const fs = require('fs');
+
+console.log("rewriting package json");
+
+const packageJson = JSON.parse(fs.readFileSync('./package.json'));
```
Comment (heatherlp) [inline .travis/rewritePackageJson.js]: I think this needs a type

File: .travis/rewritePackageJson.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+packageJson.actualActivationEvents.onView.forEach((event) => {
+  packageJson.activationEvents.push('onView:' + event);
+});
+
+packageJson.actualActivationEvents.onCommand.forEach((event) => {
+  packageJson.activationEvents.push('onCommand:' + event);
+});
+
+const packageJsonString = JSON.stringify(packageJson, null, 4);
```
Comment (heatherlp) [inline .travis/rewritePackageJson.js]: This needs a type too

File: client/test/extension.test.ts [hunk 3]
```diff
@@ -111,4 +120,65 @@ describe('Extension Tests', () => {
...
+        mySandBox.stub(vscode.commands, 'registerCommand');
+        mySandBox.stub(dependencyManager, 'hasNativeDependenciesInstalled').returns(true);
+        const installStub = mySandBox.stub(dependencyManager, 'installNativeDependencies').resolves();
+
+        const context: vscode.ExtensionContext = ExtensionUtil.getExtensionContext();
+        await myExtension.activate(context);
+        installStub.should.not.have.been.called;
+    });
+
+    it('should handle any errors', async () => {
```
Comment (heatherlp) [inline client/test/extension.test.ts]: Should this be more specific? i.e. "should handle any errors when installing dependencies"

File: client/test/extension.test.ts [hunk 4]
```diff
@@ -111,4 +120,65 @@ describe('Extension Tests', () => {
...
+        mySandBox.stub(dependencyManager, 'hasNativeDependenciesInstalled').returns(false);
+        mySandBox.stub(dependencyManager, 'installNativeDependencies').rejects({message: 'some error'});
+
+        const context: vscode.ExtensionContext = ExtensionUtil.getExtensionContext();
+        await myExtension.activate(context);
+
+        showErrorStub.should.have.been.calledWith('Failed to activate extension', 'open output view');
+    });
+
+    it('should handle any errors and show output log', async () => {
```
Comment (heatherlp) [inline client/test/extension.test.ts]: see comment above

File: .travis/rewritePackageJson.js (hunk 1 above)
Comment (caroline-church) [reply .travis/rewritePackageJson.js]: its not a typescript file its just javascript

**Pull request merged**